CMakeLists.txt

```diff
@@ -324,7 +339,7 @@ endif()
 
 add_subdirectory( plugins )
 
-if ( TRACCC_BUILD_EXAMPLES )
+if ( TRACCC_BUILD_BENCHMARK OR TRACCC_BUILD_EXAMPLES )
```

Comment:
From what I can see, you are only turning on the build of the examples when `TRACCC_BUILD_BENCHMARK` is `TRUE`, to re-use the `find_package(Boost)` call from this block. :confused:

You should just call `find_package(Boost)` in the `if(TRACCC_BUILD_BENCHMARK)` block as well. Let's not turn on the build of unnecessary things. :thinking:

Reply:
Right.. Will fix it